gandalf.js

```diff
@@ -1,121 +1,135 @@
-const formattedDate = new Date().toLocaleDateString();
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.